<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="2">
       <widget class="QLabel" name="label_lineEdit">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>radio_0</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_lineEdit_double">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="lineEdit_int"/>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>checkbox</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>radio_1</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_radio">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QSlider" name="slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_lineEdit_int">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="lineEdit_double"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_comboBox">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_checkbox">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
      </item>
      <item row="6" column="0">
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_textEdit">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QLabel" name="label_slider">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_spinBox">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="12" column="2">
       <widget class="QLabel" name="label_doubleSpinBox">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_addresult">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="edit_add1"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="edit_add2"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_key">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QSlider" name="slider_watch">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_percent">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>477</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gui.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
